task/aws/task.go

```diff
@@ -93,145 +93,145 @@ type Task struct {
 }
 
 func (t *Task) Create(ctx context.Context) error {
-	logrus.Debug("Creating DefaultVPC...")
+	logrus.Info("Creating DefaultVPC...")
```

Comment:
We excluded those logs in the past because they were too noisy in combination with the prefixes added by Terraform Core.

While they take a considerable amount of vertical space, they serve to explain why operations take too much time and give a certain sense of activity that would be lost otherwise.